internal/service/scheduler/snapshot/operator_profit/operator_profit.go

```diff
@@ -24,7 +25,7 @@ import (
 	"go.uber.org/zap"
 )
 
-var (
+var ( // FIXME: Name should be "operation_pool", update the file naem too.
```

Comment:
I am keeping the file name unchanged so we can have a diff view. It eventually needs to be changed.

Reply:
Same for the db table name, search for `// FIXME`